### DISCUSSION TITLE
Why kedacore/external-scaler-samples repository has been archived by the owner and It is now read-only.

### DISCUSSION THREAD

**Discussion by itonyli** [kedacore/keda#3157] [Q&A / Need Help]
The [samples](https://github.com/kedacore/external-scaler-samples) is incomplete，eg:

https://github.com/kedacore/external-scaler-samples/blob/c3dfac577d2374760fede6a5f36b2244698d7ec1/2.0/golang/main.go#L109

**Comment by tomkerkhove** [comment]
It is archived given it's no longer actively maintained so we want to be clear to end-users to be careful.
